config/environments/development.rb

```diff
@@ -1,6 +1,4 @@
 Rails.application.configure do
-  # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
```

Comment:
I've not read up on webpacker / yarn very much; is it ok to take this out? I'm not sure if we opted-in with it or if this was just default Rails stuff.